odoo/addons/base/models/ir_actions.py

```diff
@@ -533,6 +533,7 @@ def _default_update_path(self):
         ('set', 'Setting it to'),
         ('clear', 'Clearing it')
     ], string='Many2many Operations', default='add')
+    update_boolean_value = fields.Selection([('true', 'Yes (True)'), ('false', "No (False)")], string='Boolean Value', default='true')
```

Comment:
We could maybe create a widget later to represent boolean field as selection in the frontend ?